src/main/java/com/assignment2/Runner.java

```diff
@@ -0,0 +1,95 @@
+package com.assignment2;
+
+import com.assignment2.enums.Option;
+import com.assignment2.exceptions.GenericApplicationException;
+import com.assignment2.processing.PersistorUsingFileIO;
+import com.assignment2.processing.Processor;
+import com.assignment2.processing.interfaces.Persistent;
+import com.assignment2.readerwriter.ReaderCLI;
+import com.assignment2.readerwriter.Utility;
+import com.assignment2.readerwriter.WriterCLI;
+import com.assignment2.readerwriter.interfaces.ReaderStudentEntity;
+import com.assignment2.readerwriter.interfaces.WriterStudentEntity;
+import com.assignment2.student.Student;
+import com.assignment2.student.StudentDTO;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public final class Runner {
+
+  private static final Processor processor = new Processor();
+  private static final Utility UTILITY = new Utility();
+  private static final Persistent persistor = new PersistorUsingFileIO();
+  private static final ReaderStudentEntity reader = new ReaderCLI();
+  private static final WriterStudentEntity writer = new WriterCLI();
+
+  private Runner() {
+  }
+
+  public static void main(final String[] args) throws IOException {
+
+    ArrayList<Student> students;
+    if (persistor.testConnectivity(
+            "src\\main\\java\\com\\assignment2\\StudentsDB.txt")) {
+      students = persistor.load();
+    } else {
+      students = new ArrayList<>();
+    }
```

Comment:
testConnectivity() should be abstracted into load()

Reply:
Abstracted this connectivity testing functionality.